src/__stories__/empty-state-story.tsx

```diff
@@ -9,29 +9,44 @@ export default {
         fullScreen: true,
     },
     argTypes: {
-        asset: {
-            options: ['Icon', 'Image', 'img'],
+        actions: {
+            options: ['button and link', 'button', 'link'],
             control: {type: 'select'},
         },
     },
+    args: {
+        actions: 'button and link',
+    },
 };
 
-type Args = {asset: string};
+interface Args {
+    actions: 'button and link' | 'button' | 'link';
```

Comment:
Is having an action mandatory?
- If it's mandatory, I'd review the component props types
- If it isn't mandatory, I'd add a `'none'` option here and conditionally render the `<ButtonGroup/>` in the component

Reply:
Added `none` as option